Source/inv.cpp

```diff
@@ -2050,6 +2042,19 @@ bool UseStaff()
 	return CanUseStaff(myPlayer.InvBody[INVLOC_HAND_LEFT], myPlayer._pRSpell);
 }
 
+const Item &GetInventoryItem(int location)

```

Comment:
I think this function can take a player reference to make its purpose a bit clearer. This would match the way `UseInvItem` is called.